rebar.config

```diff
@@ -24,12 +24,12 @@
 
 {deps, [
         {leo_cache,             ".*", {git, "https://github.com/leo-project/leo_cache.git",             {tag, "0.6.5"}}},
-        {leo_commons,           ".*", {git, "https://github.com/leo-project/leo_commons.git",           {tag, "1.1.2"}}},
+        {leo_commons,           ".*", {git, "https://github.com/leo-project/leo_commons.git",           {branch, "develop"}}},
         {leo_logger,            ".*", {git, "https://github.com/leo-project/leo_logger.git",            {tag, "1.1.6"}}},
         {leo_object_storage,    ".*", {git, "https://github.com/leo-project/leo_object_storage.git",    {branch, "1.4"}}},
         {leo_redundant_manager, ".*", {git, "https://github.com/leo-project/leo_redundant_manager.git", {tag, "1.9.15"}}},
         {leo_statistics,        ".*", {git, "https://github.com/leo-project/leo_statistics.git",        {tag, "1.1.6"}}},
-        {leo_s3_libs,           ".*", {git, "https://github.com/leo-project/leo_s3_libs.git",           {tag, "1.1.8"}}},
+        {leo_s3_libs,           ".*", {git, "https://github.com/windkit/leo_s3_libs.git",               {branch, "sign_v4"}}},
```

Comment:
As your PR on leo_s3_libs was merged into 1.4 https://github.com/leo-project/leo_s3_libs/tree/1.4,
Please change to {git, "https://github.com/leo-project/leo_s3_libs.git",           {branch, "1.4"}